theano/tensor/opt.py

```diff
@@ -4584,6 +4539,10 @@ def local_fill_cut(node):
     new_inputs = []
     new = False
     for input in node.inputs:
+        # Julian: no matter what kind of function I create,
+        #         it seems that input.owner.op == T.fill is never true.
+        #         Somehow the fill ops are replaced by other ops (e.g. Elemwise{second,no_inplace}).
+        #         If that's true, I don't think we have any tests for this opt.
```

Comment:
We have tests for `test_local_fill_useless` in `theano/tensor/tests/test_op.py`, and they still pass.